manifests/clusterlogging-operator/clusterloggings.crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterloggings.logging.openshift.io
+spec:
```

Comment:
Should add [CRD validation using OpenAPI v3](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation).

Reply:
Does this look complete enough now? I have two areas commented out under `collection` because we haven't implemented them yet but have the spec created with placeholders